cdc/sink/mq_flush_worker_test.go

```diff
@@ -176,6 +176,8 @@ func TestBatch(t *testing.T) {
 	batch := make([]mqEvent, 3)
 	for _, test := range tests {
 		t.Run(test.name, func(t *testing.T) {
+			t.Parallel()
```

Comment:
We should not turn on parallel here because batch was reused and the worker itself is not thread-safe.